lib/utils/misc.js

```diff
@@ -28,5 +28,17 @@ module.exports = {
   extractRepoName(url) {
     let pathItems = url.replace(/\/$/, '').split('/')
     return pathItems.pop().replace(/\.git$/, '')
+  },
+
+  extractGithubRepo(url) {
+    if (
```

Comment:
`else { throw .. }` ?

Reply:
I chose not to throw when this is not a GH repo, as the behaviour of the user of this function (https://github.com/peopledoc/peon/pull/21/files#diff-846bea01946b5e6c06746e6e0429b58cR20) is to not push updates when it is not a GH repo. The idea being that we can watch/catch webhooks on non-GH repositories and there will just not be any status updates pushed to them.